harmoni_detectors/harmoni_stt/src/harmoni_stt/deepspeech_client.py

```diff
@@ -106,7 +107,7 @@ def is_streaming(self):
 
 
 if __name__ == "__main__":
-    model_dir = os.path.abspath(os.path.join(os.getcwd(), "../../../../../../model/deepspeech/models"))
+    model_dir = os.path.abspath(os.path.join(os.getcwd(), "../../../../harmoni_models/stt/"))
```

Comment:
Not needed for this work right now, but if you want to make this a proper standalone file/library, you would likely use `argparse` or similar to take in the model path as a user argument.